target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,121 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hootoo,tm05", "ralink,mt7620n-soc";
+	model = "HooToo TM05";
+
+	aliases {
+		label-mac-device = &ethernet;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "tm05:green:wifi";
+			gpios = <&gpio2 0 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy0radio";
+		};
+
+		power {
+			label = "tm05:blue:power";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&gpio3 {
+	status = "okay";
+};
+
+&i2c {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <80000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			factory: partition@40000 {
+				label = "factory";
+				reg = <0x40000 0x10000>;
+				read-only;
+			};
+
+			partition@50000 {
+				compatible = "denx,uimage";
+				label = "firmware";
+				reg = <0x50000 0x7b0000>;
+			};
+		};
+	};
+};
+
+&ehci {
+	status = "okay";
+};
+
+&ohci {
+	status = "okay";
+};
+
+&ethernet {
+	mtd-mac-address = <&factory 0x28>;
+	mediatek,portmap = "wllll";
```

Comment:
Since this is a one-port, I assume this is copy/paste?

Reply:
Sorry, not quite sure what you mean. Is this OK, or needing an update? Thanks!

Reply:
There is no meaning in setting wan for a port that's most probably not used anyway. I think it would be best to remove the line.

Reply:
Sure, will do - NP. Just the mac-address line, right?

Reply:
BTW, the ethernet port is on the LAN side - so this stays then, no?

Reply:
"wllll" means that you have a switch where port 0 is WAN and ports 1 to 4 are LAN.
Since you have only one port and that one is _not_ WAN by default, just remove the mediatek,portmap, since you have no information about a WAN port to pass here.

Reply:
OK, that makes sense ... ;-). Updated, built and confirmed all OK - committed (and pushed! ...LOL). Thanks!

Reply:
And re-squashed!